src/layouts/ReviewRequest/components/Comments/SendCommentForm.tsx

```diff
@@ -21,13 +22,17 @@ export interface CommentFormProps {
 export const SendCommentForm = ({
   siteName,
   requestId,
-}: CommentProps): JSX.Element => {
+  refetchData,
+}: CommentProps & {
+  refetchData: () => Promise<QueryObserverResult<CommentData[]>>
```

Comment:
could we rename this to `getComments`? 

sidenote (not v impt to fix): does using the original type of the service function not work? ie, `() => Promise<CommentData[]>`

Reply:
Removed in https://github.com/isomerpages/isomercms-frontend/pull/1133/commits/38bd4010d56b9540ee4799e69569899935692c63!

On the sidenote, the original type doesn't actually work for some reason - possibly because the refetch method is from react-query?